doc/esmvalcore/datafinder.rst

```diff
@@ -1,7 +1,227 @@
+parameter the user tells the tool what type of root paths they need the data from, e.g.:
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: BADC
+
+will tell the tool that the user needs data from a repository structured according to the BADC DRS structure `ie`
+
+``ROOT/[institute]/[dataset_name]/[experiment]/[ensemble]/[mip]/[variable_short_name]/[grid]``;
+
+setting the ``ROOT`` parameter is explained below. This is a strictly-structured repository tree and if
+there are any sort of irregularities (e.g. there is no ``[mip]`` directory) the data will not be found!
+``BADC`` can be replaced with ``DKRZ`` or ``ETHZ`` depending on the existing ``ROOT`` directory structure.
+
+The snippet
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: default
+
+is another way to retrieve data from a ``ROOT`` directory that has no DRS-like structure; ``default`` is
+a directory that contains all the needed data files (a bucket full of everything).
+
+.. note::
+   When using ``CMIP6: default`` or ``CMIP5: default`` it is important to remember that all the needed files
+   must be in the same top-level directory set by ``default`` (see below how to set ``default``).
+
+.. _config-user-rootpath:
+
+Explaining ``config-user/rootpath:``
+------------------------------------
+
+``rootpath`` identifies the root directory for different data types (``ROOT`` as we used it above):
+
+* ``CMIP`` e.g. ``CMIP5`` or ``CMIP6``: this is the `root` path(s) to where the CMIP files are stored;
+  it can be a single path or a list of paths; it can point to an ESGF node or it can point to a user
+  private repository;
+
+  Example for a CMIP5 root path pointing to the ESGF node on CEDA-Jasmin (formerly known as BADC):
+
+  .. code-block:: yaml
+
+    CMIP5: /badc/cmip5/data/cmip5/output1
+
+  Example for a CMIP6 root path pointing to the ESGF node on CEDA-Jasmin (formerly known as BADC):
+
+  .. code-block:: yaml
+
+    CMIP6: /badc/cmip6/data/CMIP6/CMIP
+
+  Example for a mix of CMIP6 root path pointing to the ESGF node on CEDA-Jasmin (formerly known as BADC)
+  and a user-specific data repository for extra data:
+
+  .. code-block:: yaml
+
+    CMIP6: [/badc/cmip6/data/CMIP6/CMIP, /home/users/joepesci/cmip_data]
+
+* ``OBS``: this is the `root` path(s) to where the observational datasets are stored; again, this could
+  be a single path or a list of paths, just like for CMIP data.
+
+  Example for the OBS path for a large cache of observation datasets on CEDA-Jasmin:
+
+  .. code-block:: yaml
+
+    OBS: /group_workspaces/jasmin4/esmeval/obsdata-v2
+
+* ``default``: this is the `root` path(s) to where files are stored without any DRS-like directory
+  structure; in a nutshell, this is a single directory that should contain all the files needed by the
+  run, without any sub-directory structure.
+
+* ``RAWOBS``: this is the `root` path(s) to where the raw observational data files are stored; this is
+  used by ``cmorize_obs``.
```

Comment:
Add link to respective section.

Reply:
no idea where that is, can you add it pls? :beer:

Reply:
https://esmvaltool.readthedocs.io/en/latest/getting_started/inputdata.html#observations

Reply:
I believe that'll throw an error since that file is not in the toc tree, we can add a ref to that after this is merged

Reply:
@JaroCamphuijsen is there a way to add a reference in ESMValCore to a section which is in ESMValTool (in this case Input data in Getting started)?